articles/overview/understanding-quantum-computing.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Understanding quantum computing
+description: What are quantum computers and how do they use the principles of quantum physics?
+author: bradben
+ms.author:  bradben
+ms.date: 04/22/2020
+ms.topic: overview
+uid: microsoft.quantum.overview.understanding
+---
+
+# Understanding quantum computing
+
+Quantum computing uses the principles of quantum physics to process information. Because of this, quantum computing requires a different approach than classical computing.  One example of this is the processor used in quantum computers.  Where classical computers use familiar silicon-based chips, quantum computers use quantum particles such atoms, ions, photons or electrons.  
+
+These quantum particles behave according to the laws of quantum physics, leveraging concepts such as probabilistic computation, superposition, and entanglement, which provide the basis for quantum algorithms that harness the power of quantum computing to solve complex problems. This article describes some of the essential concepts of quantum physics on which quantum computing is based.
+
+## A bird’s-eye view of quantum physics
+Quantum physics, also called quantum mechanics or quantum theory, is a branch of physics that deals with particles at the atomic and subatomic level. At the quantum level, however, many of the laws of physics you take for granted don’t apply. Superposition, quantum measurement, and entanglement are three phenomena that are central to quantum computing.  
+
+### Superposition vs. binary computing
+Anyone with a hectic work or school schedule knows that you can’t be two places at once – you have to be either *here* or *there*. However, if you’re a quantum particle then you can be *here* AND *there* AND *somewhere else* due to a phenomenon known as **superposition** (also known as **coherence**).
+
+Think of flipping a coin in the air and catching it. While it’s in the air turning over and over, it’s neither heads nor tails, but can be seen as a linear combination of all of its possible states at once – heads, tails, 75% heads, 12.5% tails, and so on.
+
+A quantum particle such as an electron has its own “heads or tails” property called **spin**, which can be either up or down (or positive or negative for a photon), or to make it more relatable to classical binary computing, let’s just say 1 or 0. When a quantum particle is in a superposition state, it’s a linear combination of all of its possible states between 1 and 0, but you don’t know which one it will be until you actually look at it, which brings up our next phenomenon, **quantum measurement**.
```

Comment:
Superposition need not be between all computational basis states; for instance, a two-qubit register can be in the state (|00⟩ + |11⟩) / √2, which has no support on the |01⟩ or |10⟩ basis states.